rbac/server/api/main.py

```diff
@@ -26,9 +23,9 @@
 from sanic_cors import CORS
 from zmq.asyncio import ZMQEventLoop
 
+from rbac.common.logs import getLogger
+from rbac.common.config import get_config
```

Comment:
For future, since we control these, can we standardize on snake_case or camelCase? I don't really care which, just being consistent.